docs/05-technical-reference/00-architecture/evnt-01-architecture.md

```diff
@@ -26,11 +26,11 @@ This diagram explains the event flow in Kyma, from the moment the Application se
 
 ## Event Publisher Proxy
 
-Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservices or Functions). It converts any legacy events to Cloud Events. Then, it redirects events to the NATS server. It also fetches a list of Subscriptions for a connected Application.
+Event Publisher Proxy receives legacy and Cloud Events and publishes them to the configured eventing Backend. All the legacy events are automatically converted to Cloud Events.
```

Comment:
Was backend on purpose capitalized without the leading Eventing?
```suggestion
Event Publisher Proxy receives legacy and Cloud Events and publishes them to the configured Eventing Backend. All the legacy events are automatically converted to Cloud Events.
```

Reply:
No specific reason. Should I write them both as lowercase?